proto/service/downloader/types/downloader.proto

```diff
@@ -82,6 +82,18 @@ message ListBucketReadRecordResponse {
   int64 next_start_timestamp_us = 2;
 }
 
+// GetEndpointBySpAddressRequest is request type for the GetEndpointBySpAddress RPC method
+message GetSpByAddressRequest {
+  // bucket_name is the name of the bucket
+  string bucket_name = 1;
```

Comment:
The commentary is not consistent with the struct name. Rename to `GetSpEndpointByBucketName`?

Reply:
This is an outdated version, already renamed to be consistent.